src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_omor_the_unscarred.cpp

```diff
@@ -50,190 +50,178 @@ enum Spells
     SPELL_SUMMON_FIENDISH_HOUND  = 30707
 };
 
-class boss_omor_the_unscarred : public CreatureScript
+struct boss_omor_the_unscarred : public BossAI
 {
-    public:
-
-        boss_omor_the_unscarred() : CreatureScript("boss_omor_the_unscarred") { }
-
-        struct boss_omor_the_unscarredAI : public BossAI
+    boss_omor_the_unscarred(Creature* creature) : BossAI(creature, DATA_OMOR_THE_UNSCARRED)
+    {
+        Initialize();
+        SetCombatMovement(false);
+    }
+
+    void Initialize()
+    {
+        OrbitalStrike_Timer = 25000;
```

Comment:
Move these to EventMap  while at it please

Reply:
I will update it fully one day, but not now